sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,906 @@
+DKG, or fail to vote for a new peg wallet redeem script.  If the system freezes,
+then STX unlocks do not happen, and PoX payouts do not happen either, for as
+long as the system is frozen.  Instead, miners pay to a burn address, and the PoX
+system refuses to unlock STX.  The only way the STX can unlock and PoX payments
+resume is if the system can be thawed.  If the system remains frozen across two
+reward cycle boundaries -- i.e. the current set of Stackers remain frozen and
+the subsequent set remain frozen, then the Stacks blockchain halts.
+
+To thaw the system:
+
+* On peg-in, the system can freeze if Stackers cannot complete a WSTS signing
+  round to vote to accept, reject, or abstain from a peg-in request, and the
+peg-in requests remains pending for too long.  To thaw
+the system, the Stackers would need to come back online and acknowledge all
+pending peg-in transactions.  If the system remains frozen across a prepare
+phase, then the system can be thawed by the Stackers passing the BTC to the new
+set of Stackers, which would cancel the pending peg-in requests.
+
+* On peg-out, the system can freeze if Stackers cannot complete a timely signing
+  round to vote to process the peg-out.  To thaw the system, Stackers would need
+to come back online to either fulfill all pending peg-out requests, or NACK them.
+If the system remains frozen across a reward cycle boundary, then it
+thaws if the old Stackers are able to transfer their
+BTC to the new Stackers.  This act will cancel all pending sBTC peg-out
+requests.
+
+* On peg-transfer, the system can freeze in one of two ways:  the new Stackers
+cannot vote on a new aggregate public key and redeem script before the end of
+the reward cycle, or the old Stackers cannot hand off all of the BTC to the new
+Stackers by the start of the new reward cycle. The only way to thaw the system
+is for the new Stackers to eventually complete their vote and the old Stackers
+send over the BTC before the next PoX anchor block is mined.  If the system
+remains frozen this way, and the next PoX anchor block is mined, then the Stacks
+blockchain halts.
+
+### System Recovery from Frozen Peg
+
+In the event that the BTC gets "stuck" and Stackers are unable to access it, the
+system will support a variation of a peg-in transaction called a _donation_.
+The only difference between a peg-in and a donation is that the donation does
+not materialize sBTC.  It merely gives Stackers some BTC with which to fulfill
+peg-outs.  This feature is meant to enable Stackers to recover lost BTC and
+unfreeze the system, should the need ever arise.
+ 
+## Initial Capital Limits
+
+This SIP proposes that the total amount of sBTC accepted into this system is
+capped by a limit in the `.sbtc` contract.  The `.sbtc` contract will refuse to
+admit a peg-in if it would exceed this limit.  This limit can be increased (but
+not decreased) by Stacker vote, and it would start at a default value of 100 BTC.
```

Comment:
What's the process to change this limit by stacker vote?

Reply:
Any further context on the 100 BTC limit? As in why not 1000 or 10000 or 10?

Reply:
It's up to the implementation of `.sbtc`.  This SIP does not call for a specific means.

100 BTC is just over 1% of 70% of the TVL in PoX as of today.  We could safely bump it to about 7,500 BTC before the wallet would become worth more than 70% of the locked STX.  I'm open to alternatives and compelling arguments; my rationale for a small number is to give us some time to see how well it behaves on mainnet without incurring too much risk up front.  I'm not really worried about 70% of the stackers stealing the BTC; I'm worried about someone finding and exploiting a bug that lets them clean out the sBTC peg wallet without needing the stackers' permission.